include/odp/api/spec/packet.h

```diff
@@ -1178,6 +1178,31 @@ int odp_packet_move_data(odp_packet_t pkt, uint32_t dst_offset,
  *
  */
 
+/**
+ * Flags to control payload checksum checking
```

Comment:
Why "payload"? Wouldn't "packet checksum checking" be more accurate?

Reply:
@Bill-Fischofer-Linaro updated